assets/Config/languages.json

```diff
@@ -242,6 +242,36 @@
                     },
                     "export": "خروجی"
                 }
+            },
+            "workspace": {
+                "add": {
+                    "title": "تعریف سایت جدید",
+                    "buttons": {
+                        "next": "ادامه",
+                        "finish": "پایان"
+                    },
+                    "steps": [
+                        {
```

Comment:
because of static steps, you have to use `key:value` structure in this case. (Do it on an update)